kne/deploytwodut.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+cd "$(dirname "$0")"
+
+kne delete ./twodut_topo.pb.txt || true
+cd ..
+DOCKER_BUILDKIT=1 docker build . -t "lemming:latest"
+kind load docker-image lemming:latest --name kne
+cd -
+kne create ./twodut_topo.pb.txt 
```

Comment:
ideally modify deploy.sh to have a topo argument. i can fix that in follow up PR

Reply:
It's a bit annoying because of the `dirname`, I'll let you decide what to do there.